Mathlib/NumberTheory/NumberField/CanonicalEmbedding/NormLeOne.lean

```diff
@@ -646,12 +646,7 @@ theorem interior_paramSet :
   simp [interior_pi_set Set.finite_univ, apply_ite]
 
 theorem measurableSet_interior_paramSet :
```

Comment:
I think this lemma should be deleted, an all occurrences of `measurableSet_interior_paramSet` should be replaced by `measurableSet_interior`.